qemu/tests/win_irq_check.py

```diff
@@ -38,13 +55,32 @@ def irq_check(session, device_name, devcon_folder, timeout):
             get_irq_cmd = '%sdevcon.exe resources @"%s" | find "IRQ"' % (devcon_folder,
                                                                          hwid)
             output = session.cmd_output(get_irq_cmd)
-            irq_value = re.split(r':+', output)[1]
-            if ctypes.c_int32(int(irq_value)).value < 0:
-                test.fail("%s's irq is not correct." % device_name, logging.info)
+            irq_list = output.strip().splitlines()
+            irq_nums = len(irq_list)
+            for irq in irq_list:
+                irq_value = re.split(r':+', irq)[1]
+                if irq_nums != 1:
+                    vectors = get_vectors_fqtree()
+                    if irq_nums != int(vectors):
+                        test.fail("%s's irq nums not equal to vectors." % device_name)
+                    if ctypes.c_int32(int(irq_value)).value >= 0:
+                        test.fail("%s's irq is not correct." % device_name)
```

Comment:
maybe` int(irq_value)` is enough , not need `ctypes.c_int32(int(irq_value)).value`